crates/orchestrator/src/tests/config.rs

```diff
@@ -435,3 +455,121 @@ pub mod implement_client {
         (rpc_url, starknet_client, server)
     }
 }
+
+struct EnvParams {
```

Comment:
can it read from .env.test directly?

Reply:
read `ENV_FILE` from the environment using rust_stf library and then use that file in dotenvy